File: 18주차/이연상/JadenCase 문자열 만들기.py
# [PRG] JadenCase 문자열 만들기

def solution(s):
    answer = []
    s = s.split(" ")
    for word in s:
        if word:
            answer.append(word[0].upper() + word[1:].lower())
            print(word[1:])
        else:
            answer.append(word)
    return " ".join(answer)

File: 39주차/이연상/JadenCase 문자열 만들기.java
// [PRG] JadenCase 문자열 만들기

class Solution {
    public String solution(String s) {
        String answer = "";

        String[] sList = s.split(" ");

        for(String temp : sList)
            if(temp.length() > 0)
                answer += temp.substring(0,1).toUpperCase() + temp.substring(1).toLowerCase() + " ";
            else
                answer += " ";
        if (s.charAt(s.length() - 1) != ' ')
            answer = answer.substring(0, answer.length()-1);
        return answer;
    }
}